File: CS/DxWebApp/Controllers/HomeController.cs
using DxWebApp.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace DxWebApp.Controllers
{
    public class HomeController : Controller {
        public ActionResult Index() {
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!";

            ThemesModel themeCollection = new ThemesModel();
            if(!string.IsNullOrEmpty(Utils.CurrentTheme) && !string.IsNullOrEmpty(Utils.CurrentThemeColor))
                themeCollection = new ThemesModel() { Name = "test", Theme = (CommonThemes)Enum.Parse(typeof(CommonThemes), Utils.CurrentTheme), ThemeColor = Utils.CurrentThemeColor, Font = Utils.CurrentFont };
            else
                themeCollection = new ThemesModel() { Name = "Moderno", Theme = CommonThemes.Moderno, ThemeColor = "Blue", Font = "12px 'Times New Roman'" };

            return View(themeCollection);
        }

        [HttpPost]
        public ActionResult PostTheme([ModelBinder(typeof(MyEditorsBinder))] ThemesModel themeCollection) {
            ViewBag.Message = String.Format("Current theme: {0}, theme color: {1}, font: {2}", themeCollection.Theme, themeCollection.ThemeColor, themeCollection.Font);
            return View("Index", themeCollection);
        }

        public ActionResult CalendarPartial() {
            return PartialView("CalendarPartial");
        }
    }
}

File: VB/DxWebApp/Controllers/HomeController.vb
Imports DxWebApp.Models
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace DxWebApp.Controllers
    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!"

            Dim themeCollection As New ThemesModel()
            If Not String.IsNullOrEmpty(Utils.CurrentTheme) AndAlso Not String.IsNullOrEmpty(Utils.CurrentThemeColor) Then
                themeCollection = New ThemesModel() With { _
                    .Name = "test", _
                    .Theme = DirectCast(System.Enum.Parse(GetType(CommonThemes), Utils.CurrentTheme), CommonThemes), _
                    .ThemeColor = Utils.CurrentThemeColor, _
                    .Font = Utils.CurrentFont _
                }
            Else
                themeCollection = New ThemesModel() With { _
                    .Name = "Moderno", _
                    .Theme = CommonThemes.Moderno, _
                    .ThemeColor = "Blue", _
                    .Font = "12px 'Times New Roman'" _
                }
            End If

            Return View(themeCollection)
        End Function

        <HttpPost> _
        Public Function PostTheme(<ModelBinder(GetType(MyEditorsBinder))> ByVal themeCollection As ThemesModel) As ActionResult
            ViewBag.Message = String.Format("Current theme: {0}, theme color: {1}, font: {2}", themeCollection.Theme, themeCollection.ThemeColor, themeCollection.Font)
            Return View("Index", themeCollection)
        End Function

        Public Function CalendarPartial() As ActionResult
            Return PartialView("CalendarPartial")
        End Function
    End Class
End Namespace